source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc

```diff
@@ -16,7 +16,18 @@ The order of the options or settings shown in this section may differ depending
      -e '/^OVESETUP_CONFIG\/fqdn/d' \
      /etc/ovirt-engine-setup.conf.d/20-setup-ovirt-post.conf
 ----
-
+. Remove the apache/grafana PKI files, so that they are regenerated by `engine-setup`, if enabled, using correct values. 
+//If they are missing, that's ok:
```

Comment:
You might want to remove this comment unless it needs to be there.

Reply:
Not sure why it's a comment. It's a note to the user - if 'rm' emits errors about some files missing, that's ok.